src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/CatalogPriceRulesType.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Pricing;
+
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * This form type is needed to display custom list of Catalog Price Rules.
+ * Check src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Discounts/FormTheme/discounts.html.twig
+ * for implementation.
+ */
+class CatalogPriceRulesType extends TranslatorAwareType
+{
+    /**
+     * @var LegacyContext
+     */
+    private $legacyContext;
+
+    /**
+     * PricingType constructor.
+     *
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param LegacyContext $legacyContext
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        LegacyContext $legacyContext
+    ) {
+        parent::__construct($translator, $locales);
+        $this->legacyContext = $legacyContext;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        /**
+         * %catalog_price_rule_id% can't be used in this function, because getAdminLink adds uneeded stuff to % while creating url
```

Comment:
```suggestion
         * %catalog_price_rule_id% can't be used in this function, because getAdminLink adds unneeded stuff to % while creating url
```